app/helpers/application_formatters_helper.rb

```diff
@@ -47,15 +47,15 @@ def display_user_image(user)
 
   # Links to the given User.
   #
-  # @param [User] user The User to display.
+  # @param [CourseUser] user The CourseUser to display.
   # @param [Hash] options The options to pass to +link_to+
   # @yield The user will be yielded to the provided block, and the block can override the display
   #   of the User.
   # @yieldparam [User] user The user to display.
   # @return [String] The user-visible string, including embedded HTML which will display the
   #   string within a link to bring to the User page.
   def link_to_user(user, options = {})
-    link_path = '' # TODO: Link to the user page.
+    link_path = course_user_path(user.course, user)
```

Comment:
By right I think we want to have a user page, ie. `user_path(user)` but this has not been implemented yet. This path doesn't require the context of `course`, or `course_user`. Maybe we can retain the old code for this first?